backend/program/settings/models.py

```diff
@@ -29,6 +29,8 @@ def __setattr__(self, name, value):
 class DebridModel(Observable):
     enabled: bool = False
     api_key: str = ""
+    proxy_enabled: bool = False 
+    proxy_url: str = "" 
```

Comment:
as far as I'm aware, there is no proxy? RD goes down all the time it seems, but it comes back. It's working for me atm?

Reply:
If RD Is blocking IPs, this is an easier-to-use solution than putting the whole container behind a VPN.